Library/brew.rb

```diff
@@ -39,7 +39,7 @@ def require?(path)
     if help_flag && cmd
       break
     elsif help_flag_list.include? arg
-      help_flag = true
+      help_flag = true unless cmd && arg == "help" # No help when `<cmd> help`.
```

Comment:
I find the `unless .. &&` really hard to parse. Maybe split this into a few lines/statements?